analyses/0014-vitess/vitess-analysis.md

```diff
@@ -0,0 +1,1019 @@
+
+These processes should address documentation-specific concerns, not just parrot the
+code release procedures (although there may be many of the same steps).
+
+#### Inclusive language
+
+Search the document for non-inclusive terms, especially tier-1 and tier-2 terms per
+the [Inclusive Naming Initiative](https://inclusivenaming.org/). Replace with recommended
+language where possible.
+
+## Contributor documentation
+
+Vitess is a **graduated** project of CNCF. This means that the project should
+have [_very high_][criteria] standards for documentation.
+
+| Criterion                                 | [Rating (1-5)] |
+| ----------------------------------------- | -------------- |
+| Communication methods documented          | 3. Meets standards |
+| Beginner friendly issue backlog           | 2. Needs improvement |
+| “New contributor” getting started content | 2. Needs improvement |
+| Project governance documentation          | 4. Meets or exceeds standards |
+
+### Comments
+
+The following sections contain brief assessments of each element of the
+Contributor Documentation rubric.
+
+#### Communication methods documented
+
+**Text communication channel**: Is there a Slack/Discord/Discourse/etc. community
+and is it prominently linked from your website?
+
+Yes, the community page lists a Slack channel.
+
+**Repository link**: Is there a direct link to your GitHub organization/repository?
+
+Yes, the community page and the footer list the GitHub page for the product.
+
+**Project meetings**: Are weekly/monthly project meetings documented? Is it clear
+how someone can join those meetings?
+
+Yes, the community page lists the monthly community meetings.
+
+**Mailing lists**: Are mailing lists documented?
+
+No, there does not appear to be an email list.
+
+#### Beginner friendly issue backlog
+
+**Issue triage**: Are docs issues well-triaged?
+
+No, there does not appear to be any mechanism for prioritizing issues in the GitHub
+repo.
+
+**Good first issues**: Is there a clearly marked way for new contributors to make
+code or documentation contributions (i.e. a “good first issue” label)?
+
+Yes, there is a “good first issue” label in the website repo issues list.
+
+**Issue documentation**: Are issues well-documented (i.e., more than just a title)?
+
+Yes, issues seem to have at least a paragraph description. However, there is no issues
+template or guidelines for documenting issues, so the quality of the descriptions
+is inconsistent.
+
+**Issue freshness**: Are issues maintained for staleness?
+
+No, the oldest open issue is over 6 years old.
+
+#### New contributor getting started content
+
+**Community featured on website**: Do you have a community repository or section on
+your website?
+
+Yes, there is a Community link in the site’s menu bar, leading to the Community page.
+
+**New contributor document**: Is there a document specifically for new contributors/your
+first contribution?
```

Comment:
https://vitess.io/docs/contributing/
That's for code contributions. Should we have a page for docs contributions?

Reply:
@deepthi Yes, or add doc contribution instructions to the code contribution doc.